UKHomeOffice/sgmr-service
# SPCR-532 Add submit panel for the 'change voyage details' flow

### AC / Description of changes
- add a new submit panel with CTA's for when a user is editing an existing report
- trigger this panel if user enters the report from the CYA page
- on save changes, PATCH as normal then return user to CYA page
- on cancel changes, return user to CYA page without any PATCH

### To Test

- Create a new report but do not submit (enter Melbourne as the departure port and Dover as the arrival port)
- From the Check Your Answers page click 'change' on the 'change arrival' section
- The arrivals page should have a Save changes button and a Cancel changes button
- Change the arrival port to Portsmouth
- Click Cancel changes
- User should be returned to the Check Your Answers page
- The Arrival port should be Dover (i.e. it didn't change)
- Click on the 'change' on the 'change departures' section
- Change the departure port to Sydney
- Click Save changes
- User should be returned to the Check Your Answers page
- The Departure port should now be Sydney

### Notes

---

**Remember to merge to feature branches not `master`.**
